Networking/Networking/Remote/Remote.swift

```diff
@@ -212,14 +212,14 @@ public class Remote: NSObject {
                         let validator = request.responseDataValidator()
                         try validator.validate(data: data)
                         let parsed = try mapper.map(response: data)
-                        continuation.resume(returning: .success(parsed))
+                        continuation.resume(returning: parsed)
```

Comment:
I was going to suggest that we could replace this chunk of code using the existing helpers to convert a throwing call to a Result, and a Result to a continuation. After trying it, the code is similarly long and I believe less readable, so the current approach feels better 🤷🏽‍♂️ 

Leaving it here in case it helps for other future implementations:

```swift
continuation.resume(
    with: result.flatMap { data in
        Result {
            let validator = request.responseDataValidator()
            try validator.validate(data: data)
            return try mapper.map(response: data)
        }
    }.mapError({ error in
        DDLogError("<> Mapping Error: \(error)")
        self.handleResponseError(error: error, for: request)
        return error
    })
)
```